.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   checkout_and_install:
     docker:
-      - image: circleci/node:12
+      - image: circleci/node:15
```

Comment:
We should probably use `circleci/node:lts` (which is 14). LTS releases are [recommended](https://nodejs.org/en/about/releases/) for production applications.

Reply:
I'll swap this for `lts` and see if it builds! Right now this is an exploratory PR to see what happens in the CI process. I'll need to update the docs in at least one place (maybe more) if it passes muster in CI and this can actually be merged.